templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml

```diff
@@ -328,6 +328,20 @@ Resources:
         - ParameterKey: 'OrgPrimaryAccountId'
           ParameterValue: !Ref "AWS::AccountId"
 
+  GuardDutyDeleteDetectorRoleStack:
+    Condition: install_http_logs_source
+    Type: AWS::CloudFormation::Stack
```

Comment:
why do we need this stack if we already have GuardDutyDeleteDetectorRoleStackset which deploys at the organization root id level?

Reply:
GuardDutyDeleteDetectorRoleStackset deploys with OrganizationRootID, it will not deploy for Management Account